lang/test/org/partiql/lang/eval/EvaluatingCompilerDateTimeTests.kt

```diff
@@ -1,13 +1,42 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates.  All rights reserved.
```

Comment:
Remove the copyright date especially since it's not 2019.

Reply:
Done. Thanks.